multi-key-example/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Sorry I hadn't thought of it before, but it would be nice to have a small readme in this folder explaining what we mean by multi keys example (ie what problem are we trying to solve)

Reply:
Okay sounds good